js/TrickleButtonView.js

```diff
@@ -202,10 +203,30 @@ class TrickleButtonView extends ComponentView {
    */
   async continue() {
     const parent = this.model.getParent();
-    await controller.continue();
+    const childrenAdded = await controller.continue();
+    if (!childrenAdded) return;
```

Comment:
Scroll always needs to be called. Wrap line 208 in the inverted if rather than returning early.